Sources/TensorFlow/Operators/Math.swift

```diff
@@ -613,6 +613,19 @@ internal func _vjpFloor<T: TensorFlowFloatingPoint>(
     return (floor(x), { _ in Tensor(0).broadcasted(like: x) })
 }
 
+@inlinable
+@differentiable(vjp: _vjpSign(_:))
+public func sign<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
```

Comment:
Does this have to be `TensorFlowFloatingPoint`? `tf.sign` seems to support all dtypes.

> x: A Tensor. Must be one of the following types: bfloat16, half, float32, float64, int32, int64, complex64, complex128.

Reply:
Good catch! I just updated it.